src/Helper/OrganisationApiHelper.php

```diff
@@ -0,0 +1,192 @@
+<?php
+
+namespace Drupal\os2forms_organisation\Helper;
+
+use Symfony\Component\HttpClient\CurlHttpClient;
+
+/**
+ * Organisation API helper class.
+ */
+class OrganisationApiHelper {
+
+  /**
+   * Curl Client.
+   *
+   * @var \Symfony\Component\HttpClient\CurlHttpClient
+   */
+  private ?CurlHttpClient $client = NULL;
+
+  /**
+   * The Settings.
+   *
+   * @var \Drupal\os2forms_organisation\Helper\Settings
+   */
+  private Settings $settings;
+
+  /**
+   * Constructor.
+   */
+  public function __construct(Settings $settings) {
+    $this->settings = $settings;
+  }
+
+  /**
+   * Get curl client.
+   */
+  private function getCurlClient(): CurlHttpClient {
+    if ($this->client) {
+      return $this->client;
+    }
+    else {
+      $this->client = new CurlHttpClient();
+
+      return $this->client;
+    }
+  }
+
+  /**
+   * Get bruger informationer for bruger.
+   *
+   * @phpstan-return array<string, mixed>
+   */
+  public function getBrugerInformationer(string $brugerId): array {
+    $client = $this->getCurlClient();
+
+    $this->settings->getOrganisationApiEndpoint();
+
+    $response = $client->request('GET', $this->settings->getOrganisationApiEndpoint() . 'bruger/' . $brugerId);
```

Comment:
This code is (almost) duplicated multiple times. Create a helper function, `get`, say, to keep things [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself):

```php
private function get(string $path): Response
{
  return $this->getClient()->request('GET', $this->settings->getOrganisationApiEndpoint() . $path);
}
```

Reply:
Refactored code.